### PR TITLE
[pytest] improvements for pytest infrastructure

### PR DESCRIPTION
**Pull request by stepanblyschak**
### Description of PR

Summary:

- pytest:
  - W/A some issues with pytest-ansible plugin and provided fixtures
  - add ptf adapter package and fixture to send/receive PTF traffic controlled from pytest
  - translate fdb test to use PTF adapter
  - Divide LLDP test into two test cases
- ansible
  - add pytest_runner.yml to wrap pytest in ansible run
  - remove lldp.yml test code ansible and lldp_neighbor.yml. Instead lldp.yml will run pytest test_lldp.py

Fixes # (issue)

### Type of change

- [] Bug fix
- [x] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

#### How did you verify/test it?

```
stepanb@62c67c04c2bd:/stepanb/sonic-mgmt/tests$ py.test -k 'not platform'  --testbed=$SWITCH-t0 --testbed_file=testbed.csv --inventory=../ansible/inventory --host-pattern $SWITCH-t0 -v --ignore=ptftests
=========================================================================================================== test session starts ===========================================================================================================
platform linux2 -- Python 2.7.12, pytest-4.6.4, py-1.8.0, pluggy-0.12.0 -- /usr/bin/python
cachedir: .pytest_cache
Using --randomly-seed=1564067815
ansible: 2.0.0.2
rootdir: /stepanb/sonic-mgmt/tests, inifile: pytest.ini
plugins: randomly-1.2.3, ansible-2.0.2
collected 16 items / 11 deselected / 5 selected                                                                                                                                                                                           

test_bgp_fact.py::test_bgp_facts PASSED                                                                                                                                                                                             [ 20%]
test_lldp.py::test_lldp PASSED                                                                                                                                                                                                      [ 40%]
test_lldp.py::test_lldp_neighbor PASSED                                                                                                                                                                                             [ 60%]
test_bgp_speaker.py::test_bgp_speaker SKIPPED                                                                                                                                                                                       [ 80%]
fdb/test_fdb.py::test_fdb PASSED                                                                                                                                                                                                    [100%]

========================================================================================== 4 passed, 1 skipped, 11 deselected in 147.06 seconds ===========================================================================================
```

Verify ansible LLDP test

```
ansible-playbook test_sonic.yml -i inventory -l r-boxer-sw01 -e testbed_name=r-boxer-sw01-t0 -e topo=t0 -e testcase_name=lldp -vvvvvv
```

### CODE REVIEW

**Comment by wangxin** [comment]
Regarding tests/fdb/__init__.py, is it necessary to make the fdb testing scripts a package?

File: tests/fdb/test_fdb.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+    for vlan in vlan_table:
+        for src, dst in itertools.combinations(vlan_table[vlan], 2):
+            for src_mac, dst_mac in itertools.product(fdb[src], fdb[dst]):
+                send_recv_eth(ptfadapter, src, src_mac, dst, dst_mac)
+
+    res = duthost.command('show mac')
+    logger.debug('"show mac" output on DUT')
+    for line in res['stdout_lines']:
+        logger.debug('    {}'.format(line))
```
Comment (wangxin) [inline tests/fdb/test_fdb.py]: Package pprint is for this kind of job.

**Comment by stepanblyschak** [comment]
> Regarding tests/fdb/**init**.py, is it necessary to make the fdb testing scripts a package?

No, thanks

File: tests/fdb/test_fdb.py (hunk 1 above)
Comment (stepanblyschak) [reply tests/fdb/test_fdb.py]: Thanks, I replaced with pprint

**Pull request closed without merging**